src/components/carousel/index.jsx

```diff
@@ -186,30 +206,24 @@ const Carousel = ({
 		return () => window.removeEventListener("resize", resizeFn, false);
 	});
 
-	const childItems = Children.toArray(subComponents);
-
-	let carouselItems = childItems.map((child, index) => {
-		const viewable = index + 1 > slide - slidesToShowInView && index + 1 <= slide;
-		return child.type === Item ? cloneElement(child, { viewable }) : null;
+	useEffect(() => {
+		const handleFullscreen = () => {
+			if (document.fullscreenElement) {
+				setIsFullScreen(true);
+				emitEvent("galleryExpandEnter");
```

Comment:
Adding emitEvent here means it doesn't need to be included down below in `toggleFullScreen`. It'll cause the event to send multiple times when the user clicks the buttons for opening and closing the full screen view.

Reply:
@vgalatro, Fixed this.